pymatgen/alchemy/transmuters.py

```diff
@@ -308,7 +310,7 @@ def from_filenames(poscar_filenames, transformations=None, extend_collection=Fal
         """
         trafo_structs = []
         for filename in poscar_filenames:
-            with open(filename) as file:
+            with open(filename, encoding="utf-8") as file:
                 trafo_structs.append(TransformedStructure.from_poscar_str(file.read(), []))
         return StandardTransmuter(trafo_structs, transformations, extend_collection=extend_collection)
```

Comment:
Need a closer look: this returns its super class, which differs from method docstring.